homeassistant/components/smtp/notify.py

```diff
@@ -141,6 +141,7 @@ def connect(self):
         """Connect/authenticate to SMTP Server."""
         ssl_context = client_context() if self._verify_ssl else None
         if self.encryption == "tls":
+            ssl_context.set_alpn_protocols([])
```

Comment:
This looks like its changing the context for all instances?

Reply:
Well yes, but this line actually resets it to the default value.

Reply:
Please make a new context in the executor or save it in a named constant at import time.

Reply:
Thank you for the feedback. As httpx_client also changes the alpn_protocols, I am unsure what the best course of action is. I will first try to discuss this in https://github.com/home-assistant/core/issues/138573#issuecomment-2781281784.

Reply:
I have created a new context as per your suggestion.